autogluon/task/tabular_prediction/predictor.py

```diff
@@ -129,15 +136,21 @@ def predict_proba(self, dataset, model=None, as_pandas=False):
                 Valid models are listed in this `predictor` by calling `predictor.model_names`
             as_pandas : bool (optional)
                 Whether to return the output as a pandas object (True) or numpy array (False).
-                Pandas object is a DataFrame if this is a multiclass problem, otherwise it is a Series.
+                Pandas object is a DataFrame if this is a multiclass problem or `as_multiclass=True`, otherwise it is a Series.
+                If the output is a DataFrame, the column order will be equivalent to `predictor.class_labels`.
+            as_multiclass : bool (optional)
+                Whether to return binary classification probabilities as if they were for multiclass classification.
```

Comment:
Prefix with: "This argument is only relevant for binary classification problems:"